### ISSUE TITLE
Update map of russia according to de facto situation

### ISSUE THREAD

**Issue by dolynchuk** [nvkelso/natural-earth-vector#824]
As stated in the disputed boundaries policy https://www.naturalearthdata.com/about/disputed-boundaries-policy/
"We show who actually controls the situation on the ground".
Either policy is a lie, and needs to be removed, or map should be edited.

Most recent de facto map of russia:
https://upload.wikimedia.org/wikipedia/commons/d/d1/Russia_political_location_map_%28de-facto%29.svg

Regularly updated map of "who actually controls the situation on the ground": https://deepstatemap.live/#6/49.438/32.053

### UPDATE: GeoPandas doesn't trust this dataset anymore. 
They made a patch that brings divergence compared to `naturalearth` database 
BUG PR in GeoPandas: https://github.com/geopandas/geopandas/pull/2670
Release note: https://twitter.com/geopandas/status/1601653591781085185?s=20&t=Mp2b4z-TrJPzcNuZrlcqCg

**Comment by andunai** [comment]
The statement by NaturalEarth "We show who actually controls the situation on the ground" is flawed in its nature, since NaturalEarth chose to decide what is the objectively true "situation on the ground" rather than rely on internationally accepted borders.

I cannot comprehend what value are those maps supposed to bring since NaturalEarth basically decide whatever they want, thus rendering the entire dataset unreliable and inapplicable in every possible use case.

**Comment by anti79** [comment]
The situation on the ground is also constantly changing, every other day a village is taken or lost, and sometimes it's unclear who actually controls an area. The whole "de-facto" policy was a bad idea from the beginning.

**Comment by WarthogARJ** [comment]
Trying to be unbiased here, I think that there is an emotional POV here that is clouding the actual issue.

Three points:
(1) One must understand that the Creators of this Natural Earth resource are attempting to be neutral, and flexible in how their output is used: they specifically STATE that in their Policy Statement.
In support of this is their use of the Creative Commons Public Use Mark.
And due to the data's very nature, anyone is free to edit it to reflect what THEY believe is to be the "truth".

Therefore, anyone who disputes how a given "geographical regions" borders are classified is COMPLETELY free to adjust that to their own liking.
Bearing that in mind, and without trying to cause offence, it would seem anyone who still complains about the work someone else has done, without making that effort, is either lazy, unable to edit the data, or trying to cause trouble (as in a "troll").

(2) History teaches us that Nations often expand or contract as a result of violent actions (invasions, conquering etc). It would be very hard to find many of the current "de jure" countries whose borders haven't changed within the past 100 years, let alone the past 1,000.
So my point is, that this exact issue with some bigger/more aggressive country invading another has happened many many times.

(3) The idea of any "absolute truth" in the classifications "de jure" or "de facto" is....somewhat naive in terms of the realpolitik.
There is no definitive "black/white" answer.
Again, as mentioned in the Policy Statement, there is not even any universal measure of "de jure" for many region's status. Does one use the UN's? The WorldBank? The USA? Putin? North Korea? China?

ISO 3166 ("Codes for the representation of names of countries and their subdivisions") is likely the most consistent and complete "de jure' classification you'll find: it's based on the UN recognition of countries. According to that, and the UN's General and Special Resolutions, Crimea (c/w The Port of Sevastopol), and the 

And the idea of "de facto" is also impossible to be exact about: it's always going to be "fuzzy".
Perhaps one could use the term "Pragmatic" instead.

And have a few "tests" to carry out on a given Region to decide how it should be classified "de facto/pragmatically.

Let's say we have Country A and B, and Region C: which is claimed by both A & B.

Test 1: Can an "average" citizen from the given Country travel to Region C, and exercise the same freedoms as they can in their home country?
Test 2: Can an "average" citizen from Region C travel to the given Country and exercise the same rights as they can in Region C?

Using these Tests, one would classify Crimea as "de facto" part of Russia (but NOT "de jure" applying most jurisdictions).
And Taiwan as independent, and not part of "Mainland China".

Interestingly, there are quite a few "American" territories that do not have full "American" status if one applies these tests:
District of Columbia (restricted voting rights wrt Congress)
Puerto Rico, Guam, Virgin Islands and other Territories
  
(My own emotive input: I myself am not in favour of Putin's actions: I think he should be tried as a war criminal, no better than Hitler or Stalin.
But unfortunately unless Ukraine and allies can repulse him, he HAS got Crimea under his (smelly) foot.).

**Comment by rshopa** [comment]
@WarthogARJ you replicate the typical pro-moscowian narrative, trimmed out to look "neutral" for the Western people. A major red flag for me is a comparison with Taiwan, but at least two other issues are fundamentally wrong to undermine your position:

1 - any talks about "neutrality" are irrelevant when we deal with such horrific premise. You cannot be "neutral" towards an invasion that is explicitly aimed at an eradication of the entire nation, easily fact-checked by the documented war crimes: mass murder in Bucha, Izium, Mariupil, many other places; launching ballistic and cruise missiles at densely populated urban areas; "filtration" (concentration) camps where everyone who speaks Ukrainian or has a tatoo with Ukrainian symbolic is slaughtered; mass kidnapping and displacement of children (whose parents are still alive)... The scale of the war, its distinctive imperialistic and genocidal ideology, the amount of casualties and destruction, makes it comparable to Holocaust or Holodomor. It is NOT a conflict over abstract "disputed territory", it is a WWII-type fight for survival from a 50-mln country against a monster who want it extinct, completely. 

2 - Crimea is not "disputed", it is treated as illegally annexed or, under Ukrainian law, "temporarily occupied" (see e.g. [here](https://ukraine-eu.mfa.gov.ua/en/2633-relations/spivpracya-ukrayina-yes-u-sferi-zovnishnoyi-politiki-i-bezpeki/politika-neviznannya-yes-shchodo-timchasovo-okupovanoyi-ar-krim-ta-msevastopol)). Some international institutions, for instance, UN Human Rights Council [utilised the latter definition](https://reliefweb.int/report/ukraine/situation-human-rights-temporarily-occupied-autonomous-republic-crimea-and-city-sevastopol-ukraine-report-secretary-general-ahrc5065advance-edited-version), too.

Your appeal to the historical events and speculation over "absotute truth" are also ridiculous. You may believe in anarchy and libertarinism, yet there is an acting International Law, written by blood of millions in order to prevent the catastrophies happened in the past.

**Comment by RonaldCalitri** [comment]
This is my first HOUR on this forum and can't wait to try.
BUT - IF (I haven't looked) your community is drawing a line across Crimea, IF) That would be a violation of International Law. No Quibbles, the three principal attributes of statehood are "territoriality, legitimacy, and recognition"
Territoriality you've got, short of some explosions.
Legitimacy means acceptance by population - VERY much in question,
Recognition is the capper - one or two won't do. There is a thing called "Internationally Recognized Borders."
"IF" there is such a line, kindly remove it in obediece to the UN Charter.
Thank You

**Comment by KarelWintersky** [comment]
@rshopa 

> , easily fact-checked by the documented war crimes: mass murder in Bucha, Izium, Mariupil, many other places; launching ballistic and cruise missiles at densely populated urban areas; "filtration" (concentration) camps where everyone who speaks Ukrainian or has a tatoo with Ukrainian symbolic is slaughtered; mass kidnapping and displacement of children (whose parents are still alive)... 

Of course, you have valid prooflinks?

**Comment by rshopa** [comment]
> @rshopa
> 
> > , easily fact-checked by the documented war crimes: mass murder in Bucha, Izium, Mariupil, many other places; launching ballistic and cruise missiles at densely populated urban areas; "filtration" (concentration) camps where everyone who speaks Ukrainian or has a tatoo with Ukrainian symbolic is slaughtered; mass kidnapping and displacement of children (whose parents are still alive)...
> 
> Of course, you have valid prooflinks?

By denying this, you are a war criminal yourself. But what else to expect from a moscowian?

**Comment by KarelWintersky** [comment]
@rshopa  
Your problem is that you are guessing by the avatar and blaming people by their nickname. Write already "what to take from the rashist"

**Comment by megamster** [comment]
@RonaldCalitri Well, there's a part of Portugal that all international organizations recognize as Portuguese and is nonetheless occupied by Spain and virtually all maps show as Spanish, despite the fact that even Spain itself at one point officially accepted it as being Portuguese! I suppose you have to create all your mapping data yourself since you can't use anything that violates international law and treaties... And guess what, there are dozens of other similar examples around the world.

In conclusion, it's impracticable to have maps that stick to anything other than a de facto policy

@rshopa all the things you state about Russia apply to the example I've given here. Somehow you've only now waken up to the fact that there are dozens of such cases around the world, have existed for eternity but only now are you concerned about it and even then only about the specific case you seem to be emotionally invested on

**Comment by RonaldCalitri** [comment]
Yes, you have happened upon the open borders movement. Practically speaking
the US and Europe try to follow it.
By your lights, New York's seven million people become citizens of the EU
every summer, because 36 million tourists flood the place,

**Comment by megamster** [comment]
> Yes, you have happened upon the open borders movement. Practically speaking the US and Europe try to follow it. By your lights, New York's seven million people become citizens of the EU every summer, because 36 million tourists flood the place,

@RonaldCalitri  So, according to you, Israel's occupation of Palestinian territory is just tourism? Because that would be the most similar case. Dislodging people from their homes and bringing in people from elsewhere to replace them is just tourism... Please do explain that better...

**Comment by rshopa** [comment]
@megamster even if we ignore the fact that - compared to other examples, including the Israeli occupation of the West Bank - the moscowian invasion in Ukraine that started from Crimea in 2014 has led to the largest war in Europe since 1945, it is not myself but @nvkelso who has selectively chosen ONLY ONE issue, namely Crimea, over the whole world map, having even omitted other territories ~~de-facto~~ illegally occupied by moscow, and emphasised his support of the position of the moscowian war criminals.

I can only welcome any other requests for corrections of the improper borders (nor internationally recognised) shown by Natural Earth.

> @rshopa all the things you state about Russia apply to the example I've given here. Somehow you've only now waken up to the fact that there are dozens of such cases around the world, have existed for eternity but only now are you concerned about it and even then only about the specific case you seem to be emotionally invested on

**Comment by megamster** [comment]
@rshopa and you would end up with a fictional map of much less use for everyone. Its not at random that the standard practice basically everywhere, not just Natural Earth, is to have a de facto, as opposed to de jure policy. Little kids at school learn that as they learn to interpret maps... 

As for the rest, two wrongs don't make a right 😉

**Comment by KarelWintersky** [comment]
> and emphasised his support of the position of the moscowian war criminals.

@rshopa 
This is the place to solve technical problems, not whine about your political orientation.

In any case, even the UN actually recognized Russia's protectorate over Crimea (recognizing that Russian laws apply on the territory of Crimea), which means that it recognized Crimea as Russian territory.

Further, according to the referendums held, the regions of Donetsk and Luhansk (at least) are Russian territories.

Leave the flood that this is an "illegal occupation" or "political criminals" for those who have never been on the territory of Mariupol in 2023. Tell about this to those whose houses were not shelled by the armed forces of Ukraine. Tell this to those who love to say "you need to kill all the Russians, they are rashists"

If you are not able to discuss technical problems without slipping into rabid propaganda - leave us.

**Comment by nvkelso** [comment]
The discussion here has veered into uncharted territories, as it were. I'm locking comments on this issue for 6 months and will start blocking GitHub users who can't maintain decorum.
